app/controllers/decidim/social_crowdfunding/admin/campaigns_controller.rb

```diff
@@ -5,10 +5,31 @@ module SocialCrowdfunding
     module Admin
       class CampaignsController < Decidim::Admin::Components::BaseController
         include Decidim::Paginable
+        include Decidim::SocialCrowdfunding::HasCampaign
+
+        helper Decidim::SocialCrowdfunding::Admin::ApplicationHelper
 
         helper_method :campaigns
 
-        def index; end
+        def index
```

Comment:
maybe check some permissions here?

Reply:
Sure, I had planned to implement permissions it in a different PR (see #21)